scripts/jenkins/ardana/gerrit/build_test_package.py

```diff
@@ -194,11 +195,18 @@ def _create_test_project(self):
             meta.flush()
             print("Creating test project %s linked to project %s" %
                   (self.obs_test_project_name, self.obs_linked_project))
+            buf = StringIO()
+            sh.osc('--version', _out=buf)
+            osc_ver = buf.getvalue()
             sh.osc('-A', 'https://api.suse.de', 'api', '-T', meta.name,
                    '/source/%s/_meta' % self.obs_test_project_name)
-            sh.osc('-A', 'https://api.suse.de', 'deleterequest',
+            args = ['-A', 'https://api.suse.de', 'deleterequest',
                    self.obs_test_project_name, '--accept-in-hours', 720,
-                   '-m', 'Auto delete after 30 days.')
+                   '-m', 'Auto delete after 30 days.']
+            if version.StrictVersion(osc_ver) > version.StrictVersion('0.164.1'):
```

Comment:
./scripts/jenkins/ardana/gerrit/build_test_package.py:206:16: F821 undefined name 'version'

Reply:
I pushed the wrong commit.